universe/Effect.cpp

```diff
@@ -3250,6 +3258,8 @@ void GenerateSitRepMessage::Execute(const ScriptingContext& context) const {
         }
     }
 
+    raw_sitrep_template = raw_sitrep_template && m_affiliation == AFFIL_SELF;
```

Comment:
In an MP game these will be on the server if the player hosting has them.  They could set up a distinct server, but I got the impression most people did not.

Regarding adjusting the  raw_sitrep_template variable, that's just a flag to control how it's treated.  If the affiliation is not Self then it handles it like a standard sitrep, and will generate one with errors, to help with debugging.  Should I rename the flag to "handle_as_raw_sitrep_template"?

I suppose if the host were intentionally trying to mislead other players they could still set it up with a valid message template, but thay could already do that by modifying one of the other files already.